index.html

```diff
@@ -0,0 +1,175 @@
+      </p>
+      <ul class="list-items">
+        <li><a href=""><img src="images/twitter.svg" alt="social-icons"></a></li>
+        <li><a href=""><img src="images/Linkedin-icon.svg" alt="social-icons"></a></li>
+        <li><a href=""><img src="images/medium.svg" alt="social-icons"></a></li>
+        <li><a href=""><img src="images/github.svg" alt="social-icons"></a></li>
+        <li><a href=""><img src="images/angellist.svg" alt="social-icons"></a></li>
+      </ul>
+        <ul>
+          <li><a href=""><img src="images/Disabled.svg" alt="disabled"></a></li>
+        </ul>
+          <h1>My Recent <br> Works</h1>
+        <ul>
+          <li><a href=""><img src="images/Indicator.svg" alt="indicator"></a></li>
+        </ul>
+    </section>
+    <section class="mainproject">
+      <div class="project">
+        <div class="grid container">
+          <h2>Multi-Post Stories <br> Gain+Glory</h2>
+          <ul>
+            <li>Ruby on rails</li>
+            <li>CSS</li>
+            <li>JavaScript</li>
+            <li>HTML</li>
+          </ul>
+          <button>See Project</button>
+        </div>
+      </div>
+      <div class="project">
+        <div class="grid container">
+          <h2>Multi-Post Stories <br> Gain+Glory</h2>
+          <ul>
+            <li>Ruby on rails</li>
+            <li>CSS</li>
+            <li>JavaScript</li>
+            <li>HTML</li>
+          </ul>
+          <button>See Project</button>
+        </div>
+      </div>
+      <div class="project">
+        <div class="grid container">
+          <h2>Multi-Post Stories <br> Gain+Glory</h2>
+          <ul>
+            <li>Ruby on rails</li>
+            <li>CSS</li>
+            <li>JavaScript</li>
+            <li>HTML</li>
+          </ul>
+          <button>See Project</button>
+        </div>
+      </div>
+      <div class="project">
+        <div class="grid container">
+          <h2>Multi-Post Stories <br> Gain+Glory</h2>
+          <ul>
+            <li>Ruby on rails</li>
+            <li>CSS</li>
+            <li>JavaScript</li>
+            <li>HTML</li>
+          </ul>
+          <button>See Project</button>
+        </div>
+      </div>
+      <div class="project">
+        <div class="grid container">
+          <h2>Multi-Post Stories <br> Gain+Glory</h2>
+          <ul>
+            <li>Ruby on rails</li>
+            <li>CSS</li>
+            <li>JavaScript</li>
+            <li>HTML</li>
+          </ul>
+          <button>See Project</button>
+        </div>
+      </div>
+      <div class="project">
+        <div class="grid container">
+          <h2>Multi-Post Stories <br> Gain+Glory</h2>
+          <ul>
+            <li>Ruby on rails</li>
+            <li>CSS</li>
+            <li>JavaScript</li>
+            <li>HTML</li>
+          </ul>
+          <button>See Project</button>
+        </div>
+      </div>
+    </section>
+    <div class="aboutme">
+      <div class="topaboutme">
+        <h2>About me</h2>
```

Comment:
### OPTIONAL 
- Please make the About me heading be on one line just like the Figma design 

 Figma Design          |  Your design     
:-------------------------:|:-------------------------:
![Screenshot_23](https://user-images.githubusercontent.com/69638013/159550409-cee4ea4a-1366-4836-b7a0-bd5ff83c0d7b.png)  |  ![Screenshot_22](https://user-images.githubusercontent.com/69638013/159550366-9dae625c-f569-4c3d-9166-191ee9075d5d.png)